netbox/ipam/api/views.py

```diff
@@ -88,7 +88,7 @@ def get_serializer_class(self):
         return super().get_serializer_class()
 
     @swagger_auto_schema(method='get', responses={200: serializers.AvailablePrefixSerializer(many=True)})
-    @swagger_auto_schema(method='post', responses={201: serializers.AvailablePrefixSerializer(many=True)})
+    @swagger_auto_schema(method='post', responses={201: serializers.PrefixSerializer(many=False)})
```

Comment:
Shouldn't `many` be True here, since it's possible to create multiple prefixes simultaneously?

Reply:
@jeremystretch I did this initially, but realized after some testing that it doesn't match the request body, which is a single `PrefixLength` object and suggests that only single `Prefix` object will be returned. 
If we set `many` to be `True` here for the response, then we also need to update the request body to be list of `PrefixLength` objects. But then it doesn't really match rest of the swagger spec, since most of the POST reqeust bodies and responses in the swagger spec are defined as single objects and not list of objects. 
Have used the `available-ips` endpoint as a reference to this (https://github.com/netbox-community/netbox/blob/v2.9.1/netbox/ipam/api/views.py#L166-L168).
FYI have built NetBox from this branch and tested generating a NetBox client using `go-swagger`.